### DISCUSSION TITLE
Expected Behavior of Element centeredOn for Chem.FindPotentialStereo()?

### DISCUSSION THREAD

**Discussion by vfscalfani** [rdkit/rdkit#3605] [Q&A] [answered]
Hi, I'm not sure if what I am seeing is the expected behavior for the Chem.FindPotentialStereo(). Consider the following example:

```
from rdkit import Chem
from rdkit.Chem import Draw
from rdkit.Chem.Draw import IPythonConsole
IPythonConsole.drawOptions.addAtomIndices = True
IPythonConsole.drawOptions.addStereoAnnotation = False
IPythonConsole.molSize = 300,300
m = Chem.MolFromSmiles("C\C(F)=C\C=C(/F)\C(=C\F)\C=C")
m
```
![m](https://user-images.githubusercontent.com/26606824/100745146-75445e00-33a4-11eb-8b12-4b125dc1bb01.png)
```
# Using Chem.FindPotentialStereo()
si = Chem.FindPotentialStereo(m)
for element in si:
    print(f'  Type: {element.type}, Which: {element.centeredOn}, Specified: {element.specified}, Descriptor: {element.descriptor} ')
```
output:

```
  Type: Bond_Double, Which: 2, Specified: Specified, Descriptor: Bond_Trans 
  Type: Bond_Double, Which: 4, Specified: Specified, Descriptor: Bond_Cis 
  Type: Bond_Double, Which: 7, Specified: Specified, Descriptor: Bond_Trans
```

I am expecting the first double bond to be centered on 1, not 2. If I loop through the bonds, the double bond is between atoms 1 and 3. So, index 2 is the numeric center, but it is the atom index for fluorine, so it does not seem right to me.

```
# Using GetStereo()
for b in m.GetBonds():
    print(b.GetBeginAtomIdx(),b.GetEndAtomIdx(),
          b.GetBondType(),b.GetStereo())
```

output:

```
0 1 SINGLE STEREONONE
1 2 SINGLE STEREONONE
1 3 DOUBLE STEREOZ
3 4 SINGLE STEREONONE
4 5 DOUBLE STEREOZ
5 6 SINGLE STEREONONE
5 7 SINGLE STEREONONE
7 8 DOUBLE STEREOE
8 9 SINGLE STEREONONE
7 10 SINGLE STEREONONE
10 11 DOUBLE STEREONONE
```
Thanks for any help understanding this behavior. I'm using the 2020_09_2 release from conda-forge on Ubuntu 18.04.

**Comment by greglandrum** [comment, marked as answer]
Hi @vfscalfani : the indices you are seeing are bond indices, not atom indices.
You can see them in molecule renderings with: `IPythonConsole.drawOptions.addBondIndices = True`

**Reply by vfscalfani** [reply]
wow. oops! Many thanks.
